actionview/lib/action_view/base.rb

```diff
@@ -185,6 +179,19 @@ def cache_template_loading=(value)
       def xss_safe? #:nodoc:
         true
       end
+
+      def with_empty_template_cache # :nodoc:
+        template_container = Module.new
+        Class.new(self) {
+          include template_container
```

Comment:
If we're creating anonymous subclasses, do we need the separate template container module? Can we use the subclass as the container directly?

Reply:
Ya, I was thinking this too.  I just refactored in to a module since that's what we *were* doing.  I'll pull the methods up in to the class.

Reply:
Fixed here 3ea2f9958677b0df627f92cb908e11ac9260dec1